assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -48,6 +49,22 @@ import {
 import { isShowing } from '../util';
 import AuthorSelection from './authorSelection';
 
+const StyledTextArea = styled(TextControl)`
```

Comment:
@swissspidy Could we be used styled-components in gutenberg. Does that add a lot of overhead to block editor?

Reply:
Doesnt it make more sense to use [edit.css](https://github.com/rtCamp/web-stories-wp/blob/1795867f6456d3db7dd1cb6c127fe22554f7f512/assets/src/web-stories-block/block/edit.css) -

Reply:
I understand, it definitely not as light as using the CSS. On quick search I found some 30 styled components in our block source. Some these are components we used from the `dashboard` and have overridden the styles in our block as needed.

You may also want to take a look at the blocks PRs #188 and #190.

@swissspidy

Reply:
Good points here. I see where both of you are coming from.

All the dashboard components rely on styled-components here, which is fine/expected at the moment.

That means styled-components will be already loaded and available, no matter what.

Hence it is fine to also use it here.

Should we ever use other components here (e.g. not relying on dashboard components), we might want to reconsider this.

For now I'd say it's OK though 👍